data/scenarios/facets.yaml

```diff
@@ -33,3 +33,24 @@ interference_with_protective_measures:
   description: Are there any signs of an attacker trying to maintain their access? (Persistence)
   questions:
     - user_logged_in_win
+
+operating_system:
+	uuid: f131c057-0c06-421d-b606-f3f410927d94
+	display_name: What are the involved operating systems?
+	description: What OperatingSystem(s) is in scope or relevant for this case.
+	questions:
+		- operating_system
+
+time_window_of_interest:
```

Comment:
Instead of a facet, the timeframe will be saved as a FacetTimeFrame DB object (it is part of the model) if on the facet level. Otherwise we will use the GenericAttribute mixin table to record this.

Take a look at the model for reference: https://github.com/google/timesketch/blob/88750ad09c295535f6664654ca630568bdb18c5f/timesketch/models/sketch.py#L952

Reply:
ack will remove